frame/randomness-collective-flip/src/lib.rs

```diff
@@ -98,40 +98,13 @@ decl_storage! {
 }
 
 impl<T: Trait> Randomness<T::Hash> for Module<T> {
-	/// Get a low-influence "random" value.
-	///
-	/// Being a deterministic block chain, real randomness is difficult to come by. This gives you
-	/// something that approximates it. `subject` is a context identifier and allows you to get a
-	/// different result to other callers of this function; use it like
-	/// `random(&b"my context"[..])`. This is initially implemented through a low-influence
-	/// "triplet mix" convolution of previous block hash values. In the future it will be generated
-	/// from a secure verifiable random function (VRF).
```

Comment:
Why was this deleted?

Reply:
Most of it is already present on the `Randomness` trait itself. Except the part about "In the future..." which was removed because the future is here :)